chips/cc26xx/src/ioc.rs

```diff
@@ -0,0 +1,116 @@
+/* IOC - IO Configuration */
```

Comment:
Prefer module comment ( `//!` rather than `/* .. */` ) here (same comment for several other files)

Reply:
I've updated and added some documentational comments to each of the cc26xx/ files. They're kind of meager as of now, but will be updated with future extensions and expanded usage :)